packages/donor/src/components/Wizard/WizardScreen.tsx

```diff
@@ -1,40 +1,103 @@
-import Stepper from "../basic/Stepper";
-import Button, { ButtonVariant } from "../basic/Button";
+import Button from "../basic/Button/ButtonV2";
+import Text from "../basic/Text";
+import styles from "./WizardScreen.module.scss";
+import { useState } from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import MobileStepper from "@material-ui/core/MobileStepper";
+import { BrowserRouter, Link } from "react-router-dom";
+import { PropTypes } from "@material-ui/core";
+import { ButtonVariant } from "../basic/Button";
 
-export interface WizardScreenProps {
-  imageUrl: string;
-  title: string;
-  content: string;
+const useStyles = makeStyles({
+  root: {
+    flexDirection: "column",
+    fontFamily: "Rubik",
+    backgroundColor: "#fff",
+  },
 
-  // Stepper props:
-  currentStep: number;
-  totalNumberOfSteps: number;
+  dot: {
+    margin: "8px",
+  },
 
-  // Action button
+  dots: {
+    margin: "30px 0",
+  },
+
+  dotActive: {
+    backgroundColor: "#A2D45E",
+  },
+});
+
+interface page {
+  imageUrl: string;
+  title: string;
+  content: { text: string; bold: boolean }[];
   buttonText: string;
   buttonVariant: ButtonVariant;
-
-  onNext?: () => void;
-  onPrev?: () => void;
-  onClick: () => void;
+  buttonColor: PropTypes.Color;
+}
+export interface WizardScreenProps {
+  pages: Array<page>;
+  currentStep: number;
+  linkRow?: { text: string; linkText: string; link: string };
+  callBack: () => void;
 }
 
 export default function WizardScreen(props: WizardScreenProps) {
+  const classes = useStyles();
+  const [currentStep, setCurrentStep] = useState(props.currentStep);
+  const onNext = () => {
+    if (currentStep < props.pages.length - 1) setCurrentStep(currentStep + 1);
+    else props.callBack();
+  };
   return (
     <div>
-      <img src={props.imageUrl} alt="image" />
-      <div>{props.title}</div>
-      <div>{props.content}</div>
-      <Stepper
-        step={props.currentStep}
-        numberOfSteps={props.totalNumberOfSteps}
-        handleNext={props.onNext}
-        handleBack={props.onPrev}
-      />
-      <Button
-        onClick={props.onClick}
-        title={props.buttonText}
-        variant={props.buttonVariant}
+      <img src={props.pages[currentStep].imageUrl} alt="" />
```

Comment:
You may want to limit the image size here

Reply:
done